packages/core/content-manager/admin/src/pages/EditView/components/DocumentActions.tsx

```diff
@@ -685,11 +666,7 @@ const PublishAction: DocumentActionComponent = ({
   };
 
   const totalDraftRelations = localCountOfDraftRelations + serverCountOfDraftRelations;
-
-  // TODO skipping this for now as there is a bug with the draft relation count that will be worked on separately
-  // see RFC "Count draft relations" in Notion
-  const enableDraftRelationsCount = false;
-  const hasDraftRelations = enableDraftRelationsCount && totalDraftRelations > 0;
```

Comment:
@Marc-Roig Is this what your are talking about? The check was turned off but I turned it on and it seemed to be working. Are you aware of a bug here or is this safe to merge?

Reply:
I believe the draft relation count is not working properly (logic) and we decided to disable it. It always count relations as draft if I recall correctly. 

But if we are still showing that message, lets keep the e2e test as well 🤷‍♂️

Reply:
The message only shows after removing / setting `enableDraftRelationsCount` to true. So if I think I'll need to set it to false again here and then comment out the check in the e2e tests since the dialog message won't show up anymore after that.

Reply:
ah ! I thought this flag was in the e2e tests. Yes, we should keep it as false here !
The backend logic needs a bit of love

Reply:
Got it! Thanks and good catch!